frontends/qt/server/Makefile.macosx

```diff
@@ -6,12 +6,13 @@ GOARCH=amd64
 GOOS=darwin
 
 darwin:
-	CGO_CFLAGS="-g ${GODARWINSECFLAGS} ${CFLAGS}" \
-	CGO_LDFLAGS="${GODARWINLDFLAGS} ${LFLAGS}" \
+	CGO_CFLAGS="-mmacosx-version-min=10.11 -g ${GODARWINSECFLAGS} ${CFLAGS}" \
+	CGO_LDFLAGS="-mmacosx-version-min=10.11 ${GODARWINLDFLAGS} ${LFLAGS}" \
+	MACOSX_DEPLOYMENT_TARGET=10.11 \
 	GOARCH=${GOARCH} CGO_ENABLED=${CGO} GOOS=${GOOS} \
-	go build -ldflags='-s' -x -mod=vendor \
+	go build -installsuffix oldermac -ldflags='-s -v' -x -mod=vendor \
```

Comment:
@jstrnbrg try without this and `CGO_CFLAGS` and `CGO_LDFLAGS` flags unchanged. so, only `MACOSX_DEPLOYMENT_TARGET=10.11 \` in this darwin target.